nemo/collections/asr/models/aed_multitask_models.py

```diff
@@ -674,6 +675,11 @@ def training_step(self, batch: PromptedAudioToTextMiniBatch, batch_nb):
 
         input_ids, labels = batch.get_decoder_inputs_outputs()
 
+        num_frames = batch.audio_lens.sum().float()
+        num_tokens = batch.prompted_transcript_lens.sum().float()
+        tot_frames = torch.as_tensor(batch.audio.numel(), device=num_frames.device, dtype=torch.float)
+        tot_tokens = torch.as_tensor(batch.prompted_transcript.numel(), device=num_frames.device, dtype=torch.float)
```

Comment:
puts logged metrics into float tensors so PTL doesn't have to do costly conversion; removes cuda sync on division with python floats in logged dict